hazelcast/src/main/java/com/hazelcast/internal/nio/tcp/TcpIpNetworkingService.java

```diff
@@ -175,6 +184,11 @@ public synchronized void start() {
 
         networking.restart();
         startAcceptor();
+
+        if (unifiedEndpointManager == null) {
+            refreshStatsFuture = metricsRegistry.scheduleAtFixedRate(refreshStatsTask, 1, SECONDS, ProbeLevel.INFO);
```

Comment:
1 second is ok, but consider a configurable interval, ie. Group Property. I would also start with a greater interval here, eg. 3 seconds, since its only for MC use.

Reply:
Thanks. I'll consider introducing a group property.

Reply:
I've introduced GroupProperty and changed default to 3 seconds. See: https://github.com/hazelcast/hazelcast/pull/15660/commits/d3554261ebd6d42342c72db18ed64680fb1e7966